app.js

```diff
@@ -13,6 +13,24 @@ const { json, urlencoded } = express;
 //Routes
 
 var app = express();
+const http = require('http').createServer(app);
+const io = require('socket.io')(http,  {
+  handlePreflightRequest: (req, res) => {
+      const headers = {
+          "Access-Control-Allow-Headers": "Content-Type, Authorization",
+          "Access-Control-Allow-Origin": req.headers.origin, //or the specific origin you want to give access to,
+          "Access-Control-Allow-Credentials": true
+      };
+      res.writeHead(200, headers);
+      res.end();
+  },
+  cors: {
+    origin: "https://example.com",
+    methods: ["GET", "POST"],
+    allowedHeaders: ["my-custom-header"],
+    credentials: true
```

Comment:
It's ok to start with examples from the internet, but we always need to adapt them to our use case - example.com and ['my-custom-header'] are just boilerplate information, for example.
Do we need both the handlePreflightRequest and cors? It's best to always test out how much code you can eliminate and go with the working solution that has the least amount of code - that keeps things cleaner and easier to maintain